openedx/core/djangoapps/user_authn/templates/user_authn/edx_ace/passwordreset/email/body.html

```diff
@@ -1,59 +1,66 @@
 {% extends 'ace_common/edx_ace/common/base_body.html' %}
-
 {% load i18n %}
 {% load static %}
 {% block content %}
 <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+
```

Comment:
It seems like this blank line is not required.

Reply:
There are many details in this template, this template is taken from the existing one, it's not the purpose of this PR to fix them, but to fix the main issue, however, applied.